mailing/sendgrid.go

```diff
@@ -44,7 +44,7 @@ func (s *sendgridEmailService) Send(ctx context.Context, sender string, recipien
 	if err != nil {
 		return err
 	}
-	if res.StatusCode != http.StatusOK {
+	if res.StatusCode > 299 {
```

Comment:
I usually see this check written down this way.

```suggestion
	if res.StatusCode >= 300 {
```